glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
All the handler's check_configs could be merged with their open callouts right? If we are going this route, then properly fix up the API and handlers for everyone.